src/components/grid-aware/Footer/NavigationBlock/NavigationBlock.module.css

```
@@ -0,0 +1,98 @@
+.bleedWrapper {
+  background-color: var(--color-deep-blue);
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.gridParent {
+  display: flex;
+  flex-direction: row;
+  grid-column: 2;
+  justify-content: space-between;
+  padding: 60px 0;
```

Looks like from the design, this should be:

```suggestion
  padding: 60px 0 80px;
```

---

ok